engine/requirements.txt

```diff
@@ -157,7 +157,7 @@ firebase-admin==5.4.0
     # via fcm-django
 flask==3.0.2
     # via slack-export-viewer
-google-api-core[grpc]==2.17.0
```

Comment:
similar to [this comment](https://github.com/grafana/oncall/pull/4799/files#r1711979686)